src/ui/map.js

```diff
@@ -1118,6 +1127,10 @@ class Map extends Camera {
      * Returns the map's Mapbox style object, which can be used to recreate the map's style.
      *
      * @returns {Object} The map's style object.
+     *
+     * @example
+     * var styleJson = map.getStyle();
```

Comment:
It would be nice to add a comment with a minimal example of the data structure this returns

Reply:
Is the `@returns {Object} The map's style object.` line above not sufficient? Maybe we should add more detail there instead?

Reply:
Thats sufficient !